nf_core/subworkflows/lint/__init__.py

```diff
@@ -230,3 +237,56 @@ def lint_subworkflow(self, swf, progress_bar, registry, local=False):
                 self.failed += warned
 
             self.failed += [LintResult(swf, *s) for s in swf.failed]
+
+    def update_meta_yml_file(self, swf):
```

Comment:
can't this be shared component code between modules and subworkflows?

Reply:
but that can be handled like `pattern` on a modules level, no?

Reply:
but we want the description of the input and output channels for the subworkflow, at the `meta.yml` level we don't know about the modules.